tests/neuro_san/client/agent_evaluator_factory.py

```diff
@@ -20,17 +21,22 @@ class AgentEvaluatorFactory:
     """
 
     @staticmethod
-    def create_evaluator(asserts: AssertForwarder, evaluation_type: str, test_key: str) -> AgentEvaluator:
+    def create_evaluator(asserts: AssertForwarder, evaluation_type: str) -> AgentEvaluator:
         """
         Creates AgentEvaluators
 
         :param asserts: The AssertForwarder instance to handle failures
         :param evaluation_type: A string key describing how the evaluation will take place
-        :param test_key: The test key describing the portion of the ChatMessage contents to evaluate
         """
         evaluator: AgentEvaluator = None
 
         if evaluation_type == "keywords":
-            evaluator = KeywordsAgentEvaluator(asserts, test_key)
+            evaluator = KeywordsAgentEvaluator(asserts, negate=False)
+        elif evaluation_type == "not_keywords":
+            evaluator = KeywordsAgentEvaluator(asserts, negate=True)
+        elif evaluation_type == "value":
+            evaluator = ValueAgentEvaluator(asserts, negate=False)
+        elif evaluation_type == "not_value":
+            evaluator = ValueAgentEvaluator(asserts, negate=True)
```

Comment:
Add factory entries for the ValueAgentEvalautor.  Also add negation tests for Keywords and Value as well.